Interoperability/python/qsharp/__init__.py

```diff
@@ -16,6 +16,15 @@
 from abc import ABCMeta, abstractproperty
 from random import randrange
 
+# CLR Imports #
```

Comment:
While I recognize PEP8's and pylint's guidance on import orderings, we *must* have the append to `sys.path` appear before `import clr`, otherwise the DLLs in this module will not be found by the CLR's assembly loader.